test/test__helper.rb

```diff
@@ -45,4 +45,12 @@ def load_it(judge, fb)
     $loog = Loog::NULL
     load(File.join(__dir__, "../judges/#{judge}/#{judge}.rb"))
   end
+
+  def test_date
+    test_time.to_date.to_s
+  end
+
+  def test_time
+    Time.now - (28 * 24 * 60 * 60)
```

Comment:
@Suban05 what is `(28 * 24 * 60 * 60)`? why 28 days?

Reply:
@Yegorov see [here](https://github.com/Suban05/fbe/blob/master/lib/fbe/regularly.rb)
Am I right?

Reply:
@Suban05 No, `days` is variable and can have any value different from 28.
For fix this [issue](https://github.com/zerocracy/judges-action/issues/271) use stub for `Time.now`.
See this [example](https://github.com/zerocracy/judges-action/blob/master/test/judges/test-quality-of-service.rb#L279)

Reply:
@Yegorov exactly, thanks!